tests/ltp/Makefile

```diff
@@ -1,11 +1,18 @@
+.PHONY: tests
 TOP=$(abspath ../..)
 include $(TOP)/defs.mak
 
 APPDIR=appdir
 APPBUILDER=$(TOP)/scripts/appbuilder
 
-TEST_FILE=ltp_enabled.txt
```

Comment:
@mikbras , @salsal97 enabled more tests in ltp_enabled.txt in PR https://github.com/deislabs/mystikos/pull/165 which is merged. Please make sure these new tests are enabled in tests/ltp/tests

Reply:
Hi Huseyin. Thank you for reviewing this. I backed out all my LTP changes and will leave this to salsal97. :)